framework/src/main/java/org/checkerframework/framework/source/AggregateChecker.java

```diff
@@ -1,5 +1,7 @@
 package org.checkerframework.framework.source;
 
+import static javax.tools.Diagnostic.Kind.ERROR;
```

Comment:
I'd prefer to keep this, if possible, because it makes all uses consistent throughout the codebase.

Using static imports was necessary in places because the codebase and its libraries contain multiple `Diagnostic` classes *and* multiple `Kind` classes, so only a static import would work.  So then I used it throughout.

I'll change it in this class, though.